L1Trigger/L1TMuonOverlap/src/AngleConverter.cc

```diff
@@ -88,6 +88,20 @@ int etaKeyWG2Code(const CSCDetId& detId, uint16_t keyWG) {
   return etaCode;
 }
 
+
+int fixCscOffsetGeom(int offsetLoc) {
+  // fix for CSC feo dependance from GlobalTag
+
+  // dump of CSC offsets for MC global tag
+  std::vector<int> offCSC = { -154, -133, -17, -4, 4, 17, 133, 146, 154, 167, 

```

Comment:
@konec 
Will merge into l1t-integration branch, but please make sure
you make another PR with the comment above this line explaining
what these numbers represent and how they are obtained.  Thanks.